src/api/models/mermaid.py

```diff
@@ -430,6 +430,10 @@ class Meta:
     def __str__(self):
         return '%s %s' % (self.site.__str__(), self.sample_date)
 
+    @property
```

Comment:
I think these project properties are readable and simple... Just wondering about `get_related_project` which we use in `TransectMethod` and other places where the instance is dynamic. My instinct is to consolidate into one definition of how to get a project from an instance, but I guess these are more readable?